### PR TITLE
Improvement: Delete loading window & run server in child process

### PR DESCRIPTION
**Pull request by wen-qing-255**

### CODE REVIEW

**Review by bbaa77770 — APPROVED** [review]

File: src/server-cli.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-/* eslint max-len: 0 */
-/* eslint no-console: 0 */
+// /* eslint max-len: 0 */
+// /* eslint no-console: 0 */
```
Comment (jane-rose) [inline src/server-cli.js:2]: check this

File: src/server/DataStorage.js [hunk 2]
```diff
@@ -67,10 +64,12 @@ class DataStorage {
      envDir;
 
      constructor() {
-         if (isElectron()) {
-             this.userDataDir = app.getPath('userData');
-         } else {
+         if (!isElectron()) {
              this.userDataDir = '.';
+         } else {
+             console.log('dataStorage.js-userDatadir', global.luban.userDataDir);
```
Comment (jane-rose) [inline src/server/DataStorage.js:70]: console

File: src/server-cli.js (hunk 1 above)
Comment (xiekun1992) [inline src/server-cli.js:2]: remove this

File: src/main.js [hunk 3]
```diff
@@ -186,75 +192,106 @@ if (process.platform === 'win32') {
...
     mainWindow = window;
     if (process.platform === 'win32') {
-        const outsideX = -999999, outsideY = -999999;
-        window.setSkipTaskbar(true);
-        window.blur();
-        window.setPosition(outsideX, outsideY, false);
+        // window.setSkipTaskbar(true);
+        const menu = Menu.buildFromTemplate(loadingMenu);
+        Menu.setApplicationMenu(menu);
+        window.setMenuBarVisibility(true);
```
Comment (xiekun1992) [inline src/main.js:202]: `window.setMenuBarVisibility(true);` is useless

**Review by xiekun1992 — APPROVED** [review]

**Review by jane-rose — APPROVED** [review]

**Pull request merged**